src/AffectedUserFlow/AffectedUserContext.tsx

```diff
@@ -41,6 +44,9 @@ export const AffectedUserProvider: FunctionComponent<AffectedUserProviderProps>
   const [hmacKey, setHmacKey] = useState<Key | null>(null)
   const [certificate, setCertificate] = useState<Token | null>(null)
   const [linkCode, setLinkCode] = useState<string | undefined>(undefined)
+  const [symptomOnsetDate, setSymptomOnsetDate] = useState<Posix | null>(
+    Date.now(),
```

Comment:
should this default to null? (probably okay either way as we'll figure out what makes sense for the data model when we wire up the networking side)

Reply:
It depends. In that case we'd probably want to have some local state on the component and then update this when the user continues. I'll probably leave for now and we can cross that bridge in the next bit of work.